lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -646,6 +646,18 @@ protected function prepare_links( $comment ) {
 			);
 		}
 
+		// Only grab one comment to verify the comment has children.
+		$comment_children = $comment->get_children( array( 'number' => 1, 'count' => true ) );
+		if ( ! empty( $comment_children ) ) {
+			$args = array( 'parent' => $comment->comment_ID );
+			$rest_url = add_query_arg( $args, rest_url( $this->namespace . '/' . $this->rest_base ) );
+
+			$links['children'] = array(
+				'href'       => $rest_url,
+				'embeddable' => true,
```

Comment:
Sounds good, I will change it. The original ticket I believe wanted it to be embeddable but it doesn't make much sense to embed a large collection, especially if we make a batch endpoint soon.